File: 백준/Bronze/10872. 팩토리얼/팩토리얼.c
#include <stdio.h>

int main() {
    int num, ans;
    scanf("%d", &num);
    ans = 1;

    if (num == 0) {
        printf("1\n");
    } else {
        for (int i = 1; i <= num; i++) {
            ans = ans * i;
        }
        printf("%d\n", ans);
    }

    return 0;
}

File: 백준/Bronze/10872. 팩토리얼/팩토리얼.java
import java.util.*;
import java.lang.*;
import java.io.*;

// The main method must be in a class named "Main".
class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int num = sc.nextInt();
        int ans = 1;

        if(num == 0){
        System.out.println(1);

        }else{
        
        for(int i=1; i<= num; i++){
            ans = ans * i;
        }
        System.out.println(ans);
        }

        
    }
}